include/sdf/parser.hh

```diff
@@ -40,22 +40,60 @@ namespace sdf
   //
   class Root;
 
-  /// \brief Init based on the installed sdf_format.xml file
```

Comment:
BTW This is a large amount of combinatorics... But I don't have a better solution.

Reply:
Yeah :(. I think there are some redundancies that we can deprecate, but we still end up with a lot of functions.